_data/team/roles/community_engagement.yml

```diff
@@ -0,0 +1,12 @@
+role: Community engagement
+tasks: 
+  - Responding to questions on Discord and mailing lists
+  - Managing and triaging conversations on Discord and mailing lists
+current_leads:
+  - Jenna Swarthout Goddard
+current_members:
+  - All core devs
+historical_members:
+  - Oliver Beckstein
+  - Irfan Alibay
```

Comment:
I think this is also all core devs historically too?

Reply:
Resolving as we still need to settle how we want to acknowledge historical contributors for now.